x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/job/results/BucketTests.java

```diff
@@ -33,7 +33,7 @@ public Bucket createTestInstance() {
     }
 
     public Bucket createTestInstance(String jobId) {
-        Bucket bucket = new Bucket(jobId, new Date(randomNonNegativeLong()), randomNonNegativeLong());
+        Bucket bucket = new Bucket(jobId, new Date(randomLongBetween(0, 3000000000000L)), randomNonNegativeLong());
```

Comment:
Would it make sense to also introduce a `#randomDate()` method similarly to `AutodetectResultProcessorIT`?